java/src/jmri/jmrit/withrottle/WiThrottleBundle.properties

```diff
@@ -54,9 +54,9 @@ ToolTipTurnoutCreation = Allow Wi-Fi devices to create turnouts
 LabelRoute = Routes
 ToolTipRoute = Allow Wi-Fi devices to control routes
 #StateUnknown     = Unknown < already in NamedBeanBundle properties
-#StateActive      = Active
-#StateInactive    = Inactive
-#StateInconsistent = Inconsistent
+StateActive      = Active
+StateInactive    = Inactive
+StateInconsistent = Inconsistent
```

Comment:
The reason they was commented out was probably because they already exists in [java/src/jmri/NamedBeanBundle.properties](https://github.com/JMRI/JMRI/blob/master/java/src/jmri/NamedBeanBundle.properties).

Instead of remove the comment, the best solution is probably to change `StateActive` to `SensorStateActive`, `StateInconsistent` to `BeanStateInconsistent`, and so on.
```
BeanStateInconsistent            = Inconsistent
BeanStateUnknown                 = Unknown
BeanStateUnexpected              = Unexpected value: {0}
BeanStateThrown                  = Thrown
BeanStateClosed                  = Closed
...
SensorStateActive                = Active
SensorStateInactive              = Inactive
```
This way, we don't need multiple translations of these properties.

Reply:
I saw that, but this has nothing to do with Sensors, so I thought that would be more confusing. But I'm no expert at this bundle stuff.

Reply:
I see.